tasks/clean.js

```diff
@@ -1,17 +1,20 @@
+const { basename, join } = require('path')
 const del = require('del')
+const slash = require('slash')
 
 const { destination } = require('./task-arguments.js')
+const cleanPath = slash(destination)
```

Comment:
Similar to Gulp's [`glob-parent` not supporting Windows paths](https://github.com/gulpjs/glob-parent#windows) we know our "clean" task's [`del@6.1.1`](https://github.com/sindresorhus/del/tree/5de56fd4afd16fc1d9a0108769e3b63ccbf97789#readme) package uses [`globby`](https://github.com/sindresorhus/globby#api) which also wants POSIX forward slashes 😭